### PR TITLE
Add note about repository move to README

### PR DESCRIPTION
**Pull request by adamgreig**
After this is merged I'll mark this repository as archived, and transfer remaining issues.

I think we'll have to close open PRs as they can't be transferred as-is.

### CODE REVIEW

**Comment by adamgreig** [comment]
I've disabled requiring `bors` to pass before allowing merging; CI is broken due to a new Rust version in a way that is fixed in the cortex-m repository. There isn't much point backporting the fix, but it should now be possible to manually click the merge button after approving.

**Review by thejpster — APPROVED** [review]

**Pull request merged**
